example/webpack.config.js

```diff
@@ -4,11 +4,16 @@
  * of the MIT license. See the LICENSE file for details.
  */
 
-const config = require('ca-webpack-config');
+const WriteFilePlugin = require('write-file-webpack-plugin');
+
+const config = require('../lib/');
```

Comment:
simply require the parent folder, no need to specify `lib`, the `package.json` file of the root package already contains the location of the main file (`"main": "lib/index.js"`).
```
const config = require('../');
```